.travis.yml

```diff
@@ -0,0 +1,38 @@
+sudo: required
+dist: trusty
+
+# install the pre-release chef-dk.  Use chef-stable-precise to install the stable release
+addons:
+  apt:
+    sources:
+      - chef-current-precise
+    packages:
+      - chefdk
+
+services: docker
+
+env:
+  matrix:
+  - INSTANCE=postgresql-ubuntu-1404
+  - INSTANCE=postgresql-ubuntu-1204
+  - INSTANCE=postgresql-centos-6
+  - INSTANCE=postgresql-centos-7
+
+# Don't `bundle install`
+install: echo "skip bundle install"
```

Comment:
Info needs to be added to the readme and the Gemfile should be locked with versions, otherwise testing locally and on travis will be different